test/device_map_test.rb

```diff
@@ -57,6 +57,26 @@
 
     # TODO: I do not have sufficient target map yet with enough disks and mixture of bios ids
     it "propose as first device disk containing /boot"
+
+    it "limit number of disks in device map to 8" do
+      # simple mock getting disks from partition as it need initialized libstorage
+      allow(Yast::Storage).to receive(:GetDiskPartition) do |partition|
```

Comment:
Looks like we have this (almost) same code in three different tests on this repo. Time for refactoring?